data/korath/korath ships.txt

```diff
@@ -387,6 +387,171 @@ ship "'olofez"
 	description "	Fighters do not come equipped with a hyperdrive. You cannot carry a fighter unless you have a ship in your fleet with a fighter bay."
 
 
+ship "Rai'alorej"
+	sprite "ship/rai-alorej"
+	thumbnail "thumbnail/rai-alorej"
+	attributes
+		category "Utility"
+		cost 81970000
+		shields 63000
+		hull 57000
+		"required crew" 523
+		"bunks" 1433
+		"mass" 2060
+		"drag" 29.6
+		"heat dissipation" .36
+		"fuel capacity" 700
+
+		"cargo space" 351
+		"outfit space" 1029
+		"weapon capacity" 431
+		"engine capacity" 309
+		weapon
+			"blast radius" 600
+			"shield damage" 9000
+			"hull damage" 4800
+			"hit force" 20000
+	outfits
+		"Banisher Grav-Turret" 4
+		"Warder Anti-Missile" 4
+		"Langrage Hyper-Heaver" 1
+		"Firelight Missile Bank"
+		"Firelight Missile" 18
+
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 2
+		"Small Heat Shunt" 2
+		"Fuel Processor" 2
+		"Korath Repeater Rifle" 523
+		"Microbot Defense Station" 18
+
+		"Afterburner (Planetary Class)"
+		"Reverser (Lunar Class)"
+		"Thruster (Planetary Class)"
+		"Steering (Planetary Class)"
+		"Scram Drive"
+
+	"reverse engine" -82 -193.5
+		zoom 0.8
+	"reverse engine" 82 -193.5
+		zoom 0.8
+	engine -99 181.5
+		zoom 0.9
+	engine 99 181.5
+		zoom 0.9
+	engine -60 167 0.8
+	engine 60 167 0.8
+	engine -29.5 195.5
+	engine 29.5 195.5
+	gun -81.5 41.5 "Firelight Missile Bank"
+		angle 270
+	turret -88.5 -150 "Banisher Grav-Turret"
+	turret 88.5 -150 "Warder Anti-Missile"
+	turret -110.5 -142 "Warder Anti-Missile"
+	turret 110.5 -142 "Banisher Grav-Turret"
+	turret 49.5 -53.5 "Langrage Hyper-Heaver"
+	turret -64 28 "Warder Anti-Missile"
+	turret 64 28 "Banisher Grav-Turret"
+	turret -64 50 "Banisher Grav-Turret"
+	turret 64 50 "Warder Anti-Missile"
+	bay "Fighter" -119.5 -179
+		angle 30
+	bay "Fighter" 119.5 -179
+		angle 330
+	explode "tiny explosion" 30
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 50
+	"final explode" "final explosion large"
+	description "Although fierce in battle and highly adaptable, Korath do not easily recover from injury. Without the aid of a surgeon or more equipment than fits into a standard escape pod, a stranded, injured Korath faces poor odds of survival. Therefore, Korath who leave their home territory have placed great importance on the construction of hospital ships that can quickly locate, recover, and treat the survivors of wrecked ships."
+	description "	The Rai'alorej-class Refuge, named for a pioneer in the field of microbot-assisted surgery, is thoroughly appointed with lifeform scanners, locator beacon receivers, ambulances, and search-and-rescue teams. Once onboard, crash victims are greeted with the finest hospital facilities, microbot treatments, and stores of medicine the Exiles have at hand. Even incubators and nurseries are available to treat the young who grow up aboard ship."
```

Comment:
> who grow up aboard ship

This is probably fine, but I'd ask some other grammar people if we need to change it to `ships` or `a ship` or something.

Reply:
I think "aboard ship" is something of a saying in and of itself, similar to "onboard."

Reply:
I personally think it's fine.

Reply:
From what I can tell by googling, it should be "aboard *a/the* ship", OR just "aboard" similar to "onboard"

Reply:
Aboard ship is an extremely common phrase.

Reply:
The Elenchus has Hai that can communicate with Korath, but the player does not have a communicator yet. That gives you an easy way to allow the player to communicate with Korath: visit Elenchus and have the Korath tweak the translator. Once you can speak with the Korath though, that causes trouble in the Wanderers campaign. You have to be careful when and how to allow that.

Reply:
I don't remember if the conversation took place here or on Discord, but, I was given to understand that the Korath on Elenchus would not be speaking a Korath language, they'd have assimilated into the society and speak as everyone else does.

In any case, that would only give you one Korath language, I intend for there to be several which partially but not completely intelligible to one another, and then only by native speakers or linguistic experts, not by First Last.

Reply:
Last I checked, the Korath at Elenchus speak something that is not Hai or Human Standard, and the Hai have to translate. They haven't assimilated in that sense, and so the story doesn't match what you've said. If it is to be as you say, someone needs to update the story.

Reply:
> Last I checked, the Korath at Elenchus speak something that is not Hai or Human Standard, and the Hai have to translate. They haven't assimilated in that sense, and so the story doesn't match what you've said. If it is to be as you say, someone needs to update the story.

You checked wrong, the Korath speak (poorly) a language in common with the colony. What they speak amongst themselves is their own business, but would have suffered a number of generations of linguistic drift from its source either way.
They are not suitable to provide an in for a translator - they'd probably need one themselves to communicate with their own brethren.